test/e2e/e2e_test.go

```diff
@@ -0,0 +1,251 @@
+// Copyright 2020 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
```

Comment:
we want to start building this on top of k8s e2e test framework. @pohly's work has made it possible/easier and PMEM-CSI uses it: https://github.com/intel/pmem-csi/tree/586ae281ac2810cb4da6f1e160cf165c7daf0d80/test/e2e

We could probably have a chat with him for the best practices.

Reply:
Sure. We have #253 for that. I consider this PR as a quick interim solution.